convert-redis-to-gemfire/pub-sub/tanzu-gemfire/build.gradle

```diff
@@ -3,27 +3,29 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-
 plugins {
-    id 'org.springframework.boot' version '2.4.1'
-    id 'io.spring.dependency-management' version '1.0.10.RELEASE'
+    id 'org.springframework.boot' version '3.1.3'
+    id 'io.spring.dependency-management' version '1.1.3'
     id 'java'
 }
 
 group = 'com.vmware.tanzu.gemfire'
 version = '0.0.1-SNAPSHOT'
-sourceCompatibility = '1.8'
+sourceCompatibility = '17'
 
 repositories {
     mavenCentral()
-}
-
-ext {
-    set('springGeodeVersion', "1.4.0")
+    maven {
+        credentials {
+            username "$gemfireRepoUsername"
+            password "$gemfireRepoPassword"
+        }
+        url = uri("https://commercial-repo.pivotal.io/data3/gemfire-release-repo/gemfire")
+    }
 }
 
 dependencies {
-    implementation 'org.springframework.geode:spring-geode-starter'
+    implementation 'com.vmware.gemfire:spring-boot-3.1-gemfire-10.0:1.0.0'
```

Comment:
What happened to the BYo GemFire effort? Seems like we a possibly missing a GemFire import?

Reply:
I was wondering about this. I tried it before adding gemfire-core and it all builds and runs without it. I want to investigate why that is since I would expect to have to include it manually.

Reply:
It seems gemfire-core is exposed through 'com.vmware.gemfire:spring-boot-3.1-gemfire-10.0:1.0.0'. I'll add the gemfire-core dependency anyway and create a ticket to stop exposing it.